.changeset/eleven-avocados-draw.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": patch
+---
+
+Test for activation staff member TC:38 should always work
```

Comment:
this is a bit broken, mention why it didn't work, what exactly have you fixed here. For now it's exactly same test copy-pasted to the separated file